src/genie/libs/parser/nxos/show_hsrp_event_history_msgs.py

```diff
@@ -0,0 +1,133 @@
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+
+# ====================================================
+# Schema for 'show hsrp internal event-history msgs'
+# ====================================================
+class ShowHsrpEventHistoryMsgsSchema(MetaParser):
+    
+    ''' Schema for "show hsrp internal event-history msgs" '''
+
+    schema = {  
+        "msgs":{
+            Any():{
+                 'date' : str,
+                 'time' : str,
+                 'proc_name' : str,
+                 'msg_type' : str,
+                 Optional('debug_msg'): str,
+                 Optional('opcode'): str,
+                 Optional('opcode_id'): str,
+                 Optional('ret_val'): str,
+                 Optional('src_sap'): str,
+                 Optional('dst_sap'): str,
+                 Optional('flags'): str,
+                 Optional('ha_seqno'): str,
+                 Optional('rr_token'): str,
+                 Optional('sync'): str
+
+            } 
+        }       
+    }
+
+import re
+
+# ====================================================
+# Parser for 'show hsrp internal event-history msgs'
+# ====================================================
+class ShowHsrpEventHistoryMsgs(ShowHsrpEventHistoryMsgsSchema):
+
+    ''' Parser for "show hsrp internal event-history msgs"'''
+
+    cli_command = 'show hsrp internal event-history msgs'
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+        
+        event_history_msgs_dict={}
+        result_dict={}
+
+        p1_1 = re.compile(r'^\[(?P<log_num>\d+)\]\s+(?P<date>\d{4} (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec) [0-9]{1,2}) (?P<time>[0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}\.[0-9]{6}) \[(?P<proc_name>[^\]]+)\] (?P<msg_type>E_DEBUG_DSF)\s+(?P<debug_msg>.+)$')
+        p1_2 = re.compile(r'^\[(?P<log_num>\d+)\]\s+(?P<date>\d{4} (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec) [0-9]{1,2}) (?P<time>[0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}\.[0-9]{6}) \[(?P<proc_name>[^\]]+)\] (?P<msg_type>E_MTS_RX|E_MTS_TX)\s+\[(REQ|NOT)\] Opc:(?P<opcode>[^,]+), Id:(?P<opcode_id>[^,]+), Ret:(?P<ret_val>.+)$')
+        p2 = re.compile(r'^\s*Src:[^\/]+\/(?P<src_sap>\d+), Dst:[^\/]+\/(?P<dst_sap>\d+), Flags:(?P<flags>.+)$')
+        p3 = re.compile(r'^\s*HA_SEQNO:(?P<ha_seqno>[^,]+), RRtoken:(?P<rr_token>[^,]+), Sync:(?P<sync>[^,]+), Payloadsize:(\d+)$')
+        #p4 = re.compile(r'^\s*Payload:\s*$')
+        #p5 = re.compile(r'^\s*.*$')
```

Comment:
Please add sample lines of which the regex match above each regex as a comment.

Please remove p4 and p5 since they are comments